### PR TITLE
Feature/setup design detail layout

### PR DESCRIPTION
**Pull request by syntialai**

### CODE REVIEW

File: features/tailormade_design_detail/build.gradle [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+dependencies {
+    implementation fileTree(dir: "libs", include: ["*.jar"])
+
+    // Libraries
+    api project(":features:tailormade_auth")
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+    implementation 'androidx.lifecycle:lifecycle-livedata-ktx:2.2.0'
```
Comment (blasiusneri) [inline features/tailormade_design_detail/build.gradle]: set version in ext

File: features/tailormade_design_detail/src/main/java/com/future/tailormade_design_detail/feature/view/DesignDetailActivity.kt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    binding = ActivityDesignDetailBinding.inflate(layoutInflater)
+    toolbar = binding.topToolbarDesignDetail
+    setContentView(binding.root)
+
+    setDesignDetailId(intent?.getStringExtra(PARAM_DESIGN_DETAIL_ID).orEmpty())
+  }
+
+  fun getId() = _designDetailId
```
Comment (blasiusneri) [inline features/tailormade_design_detail/src/main/java/com/future/tailormade_design_detail/feature/view/DesignDetailActivity.kt]: why need create custom getter setter? how about getter setter from kotlin

File: features/tailormade_design_detail/src/main/res/layout/layout_design_detail_general_info.xml [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        android:layout_height="wrap_content"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <ImageView
+            android:id="@+id/image_view_design_detail"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/dp_26"
```
Comment (blasiusneri) [inline features/tailormade_design_detail/src/main/res/layout/layout_design_detail_general_info.xml]: can it rounded to 24 or 28?

File: features/tailormade_design_detail/build.gradle (hunk 1 above)
Comment (syntialai) [reply features/tailormade_design_detail/build.gradle]: Will be removed kak

File: features/tailormade_search/build/intermediates/transforms/AndroidEntryPointTransform/debug/__content__.json [hunk 4]
```diff
@@ -0,0 +1 @@
+[{"name":"9870bbbadab0be191ed96ecec1da4c6ebf26a68b","index":0,"scopes":["PROJECT"],"types":["CLASSES"],"format":"DIRECTORY","present":true},{"name":"7bbeefadc13285a447dacf37e65482153d8f71a9","index":1,"scopes":["PROJECT"],"types":["CLASSES"],"format":"DIRECTORY","present":true},{"name":"87c7d1e02a493890a92942662d3a7619df1bdf3c","index":2,"scopes":["PROJECT"],"types":["CLASSES"],"format":"DIRECTORY","present":true}]
```
Comment (blasiusneri) [inline features/tailormade_search/build/intermediates/transforms/AndroidEntryPointTransform/debug/__content__.json]: put in git ignore

**Review by blasiusneri — APPROVED** [review]

**Pull request merged**